frontend/client/Styles/screens/manage_teams.scss

```diff
@@ -70,9 +70,10 @@ table {
         content: url('../../assets/arrow-down.svg');
         position: absolute;
         right: 10px;
-        top: 7px;
+        top: 10px;
         border-radius: 0;
         background-color: transparent;
+        pointer-events: none;
```

Comment:
@leon-i What is the logic for not having any pointer events?

Reply:
From my understand the `:after` that constitutes the down arrow was not clickable because it resided on top of the select and had no mouse events associated with it. Granting it `pointer-events: none` allows it to be clicked through, giving the desired effect.